From f24b1399d40bb905ef3407ad00bb3f85da8de9df Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Sun, 2 Jan 2022 11:38:35 +0000
Subject: [PATCH 2/5] env: fat: Add new lines at the end of print statements

Add some new line feeds at the end of print messages to make things
easier to read on the console. The other env options do this so
this is just an omission for FAT env.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
---
 env/fat.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/env/fat.c b/env/fat.c
index 9d37d26f..fdccd6cd 100644
--- a/env/fat.c
+++ b/env/fat.c
@@ -77,7 +77,7 @@ static int env_fat_save(void)
 		 * This printf is embedded in the messages from env_save that
 		 * will calling it. The missing \n is intentional.
 		 */
-		printf("Unable to use %s %d:%d... ",
+		printf("Unable to use %s %d:%d... \n",
 		       CONFIG_ENV_FAT_INTERFACE, dev, part);
 		return 1;
 	}
@@ -93,7 +93,7 @@ static int env_fat_save(void)
 		 * This printf is embedded in the messages from env_save that
 		 * will calling it. The missing \n is intentional.
 		 */
-		printf("Unable to write \"%s\" from %s%d:%d... ",
+		printf("Unable to write \"%s\" from %s%d:%d... \n",
 			file, CONFIG_ENV_FAT_INTERFACE, dev, part);
 		return 1;
 	}
@@ -135,7 +135,7 @@ static int env_fat_load(void)
 		 * This printf is embedded in the messages from env_save that
 		 * will calling it. The missing \n is intentional.
 		 */
-		printf("Unable to use %s %d:%d... ",
+		printf("Unable to use %s %d:%d... \n",
 		       CONFIG_ENV_FAT_INTERFACE, dev, part);
 		goto err_env_relocate;
 	}
@@ -153,7 +153,7 @@ static int env_fat_load(void)
 		 * This printf is embedded in the messages from env_save that
 		 * will calling it. The missing \n is intentional.
 		 */
-		printf("Unable to read \"%s\" from %s%d:%d... ",
+		printf("Unable to read \"%s\" from %s%d:%d... \n",
 			CONFIG_ENV_FAT_FILE, CONFIG_ENV_FAT_INTERFACE, dev, part);
 		goto err_env_relocate;
 	}
-- 
2.34.1

